### PR TITLE
Feature/controllers

### PR DESCRIPTION
**Pull request by Van-Ku**

### CODE REVIEW

File: OMG.Classroom/API/Properties/launchSettings.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
-﻿{
+{
```
Comment (georgipetkov18) [inline OMG.Classroom/API/Properties/launchSettings.json:1]: Why do you change a configuration file

**Review by georgipetkov18 — CHANGES_REQUESTED** [review]

File: OMG.Classroom/API/Properties/launchSettings.json [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
   "$schema": "https://json.schemastore.org/launchsettings.json",
   "iisSettings": {
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:25123",
-      "sslPort": 44336
+      "applicationUrl": "http://localhost:52293/",
```
Comment (georgipetkov18) [inline OMG.Classroom/API/Properties/launchSettings.json:7]: Why do you change a configuration file

**Review by georgipetkov18 — CHANGES_REQUESTED** [review]
Do not change the configuration file

**Comment by apriori-yarik** [comment]
Nice commit message xD

**Review by georgipetkov18 — APPROVED** [review]
I completely understood what you had to say. That message was so thorough.

**Pull request merged**